src/commonMain/kotlin/org/hisp/dhis/lib/expression/spi/AndroidCustomIntentVariable.kt

```diff
@@ -0,0 +1,12 @@
+package org.hisp.dhis.lib.expression.spi
+
+import kotlin.js.JsExport
+
+@JsExport
+enum class AndroidCustomIntentVariable {
```

Comment:
I decided to create an enum just to be able to validate the expression.

Reply:
Using an enum sure makes `VAR{...}` just usable by android (unless someone has the exact use case of one or more of these variables). It is still fine but it is somewhat of an odd mix to pair a generic name with a very specific usability. Anyhow, the expression syntax is already odd so I think we just go with what you like better. If we ever want to make something that composes better we would have to invent a 2.0 version anyhow. 

FYI: When I make this into the context key-value the key would be a wrapper on this enum or the enum itself if it can implement the interface. For the value we should talk about if you rather want a wrapper on `Any` or a wrapper having 1 field per possible return type.